node/utils/simulation.ts

```diff
@@ -0,0 +1,81 @@
+function distinct<T>(value: T, index: number, self: T[]) {
+  return self.indexOf(value) === index
+}
+
+export const getSimulationPayloads = (items: Item[], priceTable?: string, regionId?: string) => {
```

Comment:
Should sales channel be considered as well?

Reply:
It is, but it is a [querystring parameter](https://github.com/vtex-apps/store-graphql/blob/feature%2Fproduct-with-simulation/node/clients/checkout.ts#L204)